nuxt.config.js

```diff
@@ -1,4 +1,4 @@
-import VuetifyLoaderPlugin from 'vuetify-loader/lib/plugin'
```

Comment:
No more useful?

Reply:
Yes it is usefull but directly handle by nuxt with https://github.com/nuxt-community/vuetify-module